### PR TITLE
Test Shim: Move network tests into network package

### PR DESCRIPTION
**Pull request by mbfrahry**

### CODE REVIEW

File: azurerm/internal/services/network/network_security_group_resource_test.go [hunk 1]
```diff
@@ -1,17 +1,18 @@
-package tests
+package network_test
 
 import (
 	"context"
 	"fmt"
 	"testing"
 
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/network"
```
Comment (manicminer) [inline azurerm/internal/services/network/network_security_group_resource_test.go:8]: Did you mean to group this separately?

File: azurerm/internal/services/network/point_to_site_vpn_gateway_resource_test.go [hunk 2]
```diff
@@ -1,16 +1,17 @@
-package tests
+package network_test
 
 import (
 	"context"
 	"fmt"
 	"testing"
 
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/network"
```
Comment (manicminer) [inline azurerm/internal/services/network/point_to_site_vpn_gateway_resource_test.go:8]: also this?

File: azurerm/internal/services/network/route_table_resource_test.go [hunk 3]
```diff
@@ -1,17 +1,18 @@
-package tests
+package network_test
 
 import (
 	"context"
 	"fmt"
 	"testing"
 
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/network"
```
Comment (manicminer) [inline azurerm/internal/services/network/route_table_resource_test.go:8]: and this?

**Review by manicminer — APPROVED** [review]
Hey @mbfrahry, LGTM. Just noticed a couple of imports that you may have meant to order separately but I wasn't sure.

File: azurerm/internal/services/network/network_security_group_resource_test.go (hunk 1 above)
Comment (mbfrahry) [reply azurerm/internal/services/network/network_security_group_resource_test.go]: Sure didn't! Taken care of! Thanks

**Pull request merged**